packages/frontend/package.json

```diff
@@ -80,7 +80,6 @@
     "eslint-plugin-jest-dom": "^5.1.0",
     "eslint-plugin-storybook": "^0.8.0",
     "eslint-plugin-testing-library": "^6.2.0",
-    "husky": "^9.0.11",
```

Comment:
**issue (blocking):**  this might affect our pre-commit checks. Could you please clarify why it was removed?